test/external-dependencies/boost.cpp

```diff
@@ -22,12 +22,14 @@
 
 #include <boost/test/unit_test.hpp>
 #include <libdevcore/Common.h>
+#include <test/TestHelper.h>
 
 BOOST_AUTO_TEST_SUITE(ExtDepBoost)
 
 // test that reproduces issue https://github.com/ethereum/cpp-ethereum/issues/1977
 BOOST_AUTO_TEST_CASE(u256_overflow_test)
 {
+	dev::test::TestOutputHelper::initTest();
```

Comment:
I still can't imagine that boost test is supposed to work that way, that you need to `cnote << testname` ot use your helper. I don think you need to add a user defined function, just check the commandline flags or the docs (http://www.boost.org/doc/libs/1_43_0/libs/test/doc/html/utf/user-guide/test-output.html)